src/pudl/metadata/resources/eia923.py

```diff
@@ -958,6 +958,51 @@
         "sources": ["eia923"],
         "etl_group": "eia923",
     },
+    "_core_eia923__emissions_control": {
+        "description": {
+            "additional_summary_text": (
+                "Actual rate and removal efficiency for air emissions reported by "
+                "thermoelectric or combined-cycle power plants with a total steam turbine capacity "
+                "greater than or equal to 10 megawatts."
+            ),
+            "additional_source_text": "(Schedule 8C)",
+        },
+        "schema": {
+            "fields": [
+                "plant_id_eia",
+                "report_year",
```

Comment:
While the reporting frequency for this table is annual, we use `report_date` with a datetime type and January 1st as the date so the column can be used in conjunction with other datetime columns. So this should be converted to a date in the transform and renamed to `report_date`.